### PR TITLE
All Actuators Sample

### PR DESCRIPTION
**Pull request by kenswan**
- List of actuator endpoints on UI landing page
- APIs with additional requests/logging (Swagger UI available)
- Steeltoe branding/resources

### CODE REVIEW

File: Management/src/AllActuators/Actuators/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+applications:
+- name: sbo-integration
+  memory: 1024M
+  command: cd ${DEPS_DIR}/0/dotnet_publish && exec ./Steeltoe.Actuators --server.urls https://sbo-integration.apps.pcfone.io
```
Comment (TimHess) [inline Management/src/AllActuators/Actuators/manifest.yml:5]: Won't the buildpack will set this automatically?
```suggestion
```

File: Management/src/AllActuators/Actuators/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "Logging": {
```
Comment (TimHess) [inline Management/src/AllActuators/Actuators/appsettings.json:2]:
```suggestion
  "$schema": "https://steeltoe.io/schema/latest/schema.json",
  "Logging": {
```

File: Management/src/AllActuators/Actuators/appsettings.Development.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "Logging": {
```
Comment (TimHess) [inline Management/src/AllActuators/Actuators/appsettings.Development.json:2]:
```suggestion
  "$schema": "https://steeltoe.io/schema/latest/schema.json",
  "Logging": {
```

File: Management/src/AllActuators/Actuators/manifest.yml (hunk 1 above)
Comment (kenswan) [reply Management/src/AllActuators/Actuators/manifest.yml:5]: It should... I think this is leftover from some troubleshooting. I'll remove it and do a push to make sure, then update this file.

File: Management/src/AllActuators/Actuators/appsettings.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+          "Include": [ "*" ]
+        }
+      }
+    },
+    "metrics": {
+      "exporter": {
+        "cloudfoundry": {
+          "validateCertificates": false
+        }
+      }
```
Comment (TimHess) [inline Management/src/AllActuators/Actuators/appsettings.json:28]:
```suggestion
```

File: Management/src/AllActuators/Actuators/manifest.yml (hunk 1 above)
Comment (kenswan) [reply Management/src/AllActuators/Actuators/manifest.yml]: This was left over when troubleshooting why the wrong .NET version (3.1) was being targeted by default. The `buildpack.yml` file resolved that issue. This has been removed.

**Review by TimHess — APPROVED** [review]

**Pull request merged**